concord-ui/package.json

```diff
@@ -8,11 +8,11 @@
     "react-intl": "^2.4.0",
     "react-redux": "^5.0.7",
     "react-router-dom": "^4.2.2",
-    "react-scripts": "1.1.1",
-    "redux": "^3.7.2",
+    "react-scripts": "2.1.1",
+    "redux": "^4.0.1",
     "redux-thunk": "^2.3.0",
-    "rmwc": "^1.3.0",
-    "superagent": "^3.8.2",
+    "rmwc": "^3.0.10",
+    "superagent": "^4.0.0",
```

Comment:
Should we pin the versions?

Reply:
That would make sense

Reply:
Agree lets pin the versions (remove the ^) and then we can merge.